### ISSUE TITLE
WP1 selections fail to unzip

### ISSUE THREAD

**Issue by kelson42** [openzim/wp1_selection_tools#44]
Here is the error
```
  inflating: /data/tmp/enwiki_2022-04/projects/Zambia.tsv
  inflating: /data/tmp/enwiki_2022-04/projects/Zimbabwe.tsv
  inflating: /data/tmp/enwiki_2022-04/projects/Zoo.tsv
projects/Bah#U00c3#U00a1'#U00c3#U00ad_Faith.tsv:  mismatching "local" filename (projects/Bah�¡'�­_Faith.tsv),
       continuing with "central" filename version
projects/Beyonc#U00c3#U00a9.tsv:  mismatching "local" filename (projects/Beyonc�©.tsv),
        continuing with "central" filename version
projects/Chi#U00c5#U009fin#U00c4#U0083u.tsv:  mismatching "local" filename (projects/Chi�
�in�
    �u.tsv),
         continuing with "central" filename version
projects/Chi#U00c8#U0099in#U00c4#U0083u.tsv:  mismatching "local" filename (projects/Chi��in�
                                                                                             �u.tsv),
         continuing with "central" filename version
projects/Cura#U00c3#U00a7ao.tsv:  mismatching "local" filename (projects/Cura�§ao.tsv),
        continuing with "central" filename version
projects/Human#U00e2#U0080#U0093Computer_Interaction.tsv:  mismatching "local" filename (projects/Humanâ��Computer_Interaction.tsv),
        continuing with "central" filename version
projects/M#U00c4#U0081ori.tsv:  mismatching "local" filename (projects/M�
                                                                         �ori.tsv),
         continuing with "central" filename version
projects/Pok#U00c3#U00a9mon.tsv:  mismatching "local" filename (projects/Pok�©mon.tsv),
        continuing with "central" filename version
```

**Comment by rgaudin** [comment]
Ah I realize you did not include my unzip fix in your commit!

I didn't either as I considered this done.

Can you add -UU to the unzip command ?

**Comment by kelson42** [comment]
It seems I have already fixed this issue in f232aacdbad2d30093db67d1b1ab09765600fdec
